src/shared/config/AuthXmlConfigFileLocator.php

```diff
@@ -2,24 +2,37 @@
 namespace PharIo\Phive;
 
 class AuthXmlConfigFileLocator {
-    private const FILENAME = 'phive-auth.xml';
-
     /** @var Config */
     private $config;
 
     /** @var Environment */
     private $environment;
 
-    public function __construct(Environment $environment, Config $config) {
+    /** @var Cli\Output */
+    private $output;
+
+    public function __construct(Environment $environment, Config $config, Cli\Output $output) {
         $this->environment = $environment;
         $this->config      = $config;
+        $this->output      = $output;
     }
 
     public function getFile(bool $global): \PharIo\FileSystem\Filename {
         if ($global) {
-            return $this->config->getHomeDirectory()->file(self::FILENAME);
+            return $this->config->getHomeDirectory()->file('auth.xml');
```

Comment:
Same again: let's move the `file()` call into the config, e.g. `$this->config->getHomeAuthFile()` ?